app/modules/asset_groups/models.py

```diff
@@ -118,6 +119,50 @@ class AssetGroupSighting(db.Model, HoustonModel):
     # May have multiple jobs outstanding, store as Json obj uuid_str is key, In_progress Bool is value
     jobs = db.Column(db.JSON, nullable=True)
 
+    def start_sage_detection(self, model_config, detection):
+        # TODO make this a task so that it's non blocking but handle the response and
+        # inform AssetGroupSighting if it fails
+        current_app.acm.request_passthrough_result(
+            'job.detect_request', 'post', {'params': model_config}, detection
+        )
+
+    def run_sage_detection(self, model):
+        job_id = uuid.uuid4()
+        base_url = current_app.config.get('BASE_URL')
+        callback_url = f'{base_url}api/v1/asset_group/sighting/{str(self.guid)}/sage_detected/{str(job_id)}'
+        # TODO use model to build up the input, also not clear on where the endpoint & function come from
+        model_config = {
+            'endpoint': '/api/engine/detect/cnn/lightnet/',
+            'function': 'start_detect_image_lightnet',
+            'jobid': str(job_id),
+            'callback_url': callback_url,
+            'image_uuid_list': [],
+            'input': {
+                'callback_url': callback_url,
+                'image_url': f'{base_url}api/v1/asset/src-raw/',
+                'labeler_model_tag': 'iot_v0',
+                'model_tag': 'iot_v0',
+                'labeler_algo': 'densenet',
+                'sensitivity': 0.36,
+                'nms_aware': 'ispart',
+                'nms_thresh': 0.5,
+                'callback_detailed': True,
+            },
+        }
+
+        for filename in self.config.get('assetReferences'):
```

Comment:
will `assetReferences` possibly contain duplicates here?  i know we could have duplicates across _encounters_ but just checking.  no need to send an asset more than once to detection, basically.

Reply:
Good point, will check for that before appending